get-started/guide-neo4j-browser/guide-neo4j-browser.adoc

```diff
@@ -97,174 +147,111 @@ m|:help keys
-Within the GRASS file you can change colors, fonts, sizes, outlines and titles per node-label and relationship-type.
+Within the GRASS file, you can change colors, fonts, sizes, outlines, and titles per node label and relationship type.
 It is also possible to combine multiple properties into a caption with `caption: '{name}, born in {born}';`
 
-image::{img}style_sheet_grass.jpg[]
-
-
-////
-== Some Tips and Tricks for Neo4j Browser
-
-* switch to multi-line editing mode with kbd:[Shift+Enter], then kbd:[Enter] will create newlines
-* execute with kbd:[Ctrl+Enter] (kbd:[Cmd+Enter] on OSX)
-* you can click on the query text above the graph visualization to get it back into the editor
-* Navigate history with kbd:[Ctrl+Up] and kbd:[Ctrl+Up], access all of it via `:history`. The history will be persisted across browser restarts
-* `ESC` puts the editor in fullscreen mode, useful for editing large queries
-* You can switch between tabular, visual mode, query plan and x-ray mode for results with the icons on the left of each panel, 
-** don't worry if you don't see any output, you might just be in visual mode but returned tabular/scalar data, just switch the mode to tabular
-** query time is reported in the tabular view, don't rely on that timing it includes the latency and (de-)serialization costs, not just the actual query execution time
-** you can download the results as CSV from the tabular output panel (top right download icon), and as JSON (download icon above the panel)
-* `:clear` removes all accumulated output frames, the cross icon removes a single frame and kills a running query. You can also expand a panel into fullscreen (helpful in visual mode)
-
-=== Favorites
-
-* if you want to save a snippet to your favorites (using the star button), make sure to add a comment line on top (`// ...`) which will be used as title for your snipppet
-* you can group your favorites into folders and order them with drag and drop and remove them again
-////
+image::{img}style_sheet_grass.png[]
 
 == Configuration
 
-* since Neo4j 2.3 there is a config drawer on the left (with the cog), no need for the `:config` command anymore
-* you can retrieve the current configuration with `:config`
-* the individual settings are configured with:
+The defaults for all the settings can adjusted at any time by going to the configuration option on the left-side menu.
+
+Some possible config changes and views are listed below.
+* You can retrieve the current configuration with `:config`.
+* Individual settings are configured with the following defaults:
 ** `:config maxNeighbours:100` - maxiumum number of neighbours for a node
 ** `:config maxRows:100` - maximum number of rows for the tabular result
 
 == Executing REST requests
 
-You can also execute REST requests with the Neo4j Browser, the command-syntax is +
-`:COMMAND /a/path {"some":"data"}`.
+You can also execute REST requests with the Neo4j in the browser. The command syntax is `:COMMAND /a/path {"some":"data"}`.
```

Comment:
did you check that this still works? it was broken at some point in the new browser after the rewrite.

Reply:
Not sure if we need to keep the whole REST section here, as this is quite specific stuff, and probably too much for an "Intro" ?

Reply:
Oops, I forgot to test this section! Just tested now, and they do still work. I agree that most beginners probably wouldn't use it, but I'm afraid we'd get questions about it if we didn't include it. I didn't realize you could do this before I read this section. I'm tempted to leave it there. If someone's curious, at least we have the info somewhere. Don't you think?